upgrade/upgradeEtrogSovereign/upgrade_parameters.json.example

```diff
@@ -0,0 +1,19 @@
+{
+    "unsafeMode": false,
+    "bridgeL2Address": "0x..",
+    "bridge_initParams": {
+        "bridgeManager": "0x..",
+        "proxiedTokensManagerAddress": "0x..",
+        "emergencyBridgePauserAddress": "0x..",
+        "emergencyBridgeUnpauserAddress": "0x.."
+    },
+    "ger_initParams": {
+        "globalExitRootUpdater": "0x..",
+        "globalExitRootRemover": "0x.."
+    },
+    "pathTokensJson": "",
```

Comment:
remove this param and hardcode the file to retrieve data to `tokens.json`.
We save one input and error/usage is clearer